Plugins/GitUIPluginInterfaces/GitCmdResult.cs

```diff
@@ -8,25 +8,22 @@ public struct CmdResult
         public string StdError;
         public int ExitCode;
 
-        public bool ExitedSuccessfully
-        {
-            get
-            {
-                return ExitCode == 0;
-            }
-        }
+        public bool ExitedSuccessfully => ExitCode == 0;
```

Comment:
:bulb: Unrelated refactoring in this file is better suited to a separate pull request